interval/query.go

```diff
@@ -57,6 +58,8 @@ func ReadToChan(inputFile string, send chan<- Interval) {
 		for val := range receive {
 			send <- val
 		}
+	default:
+		log.Fatalf("ERROR: file type of %s not supported by interval ReadToChan")
```

Comment:
looks good to me, but looks like theres a go fmt issue that you may want to look into